### ISSUE TITLE
js error in IE10

### ISSUE THREAD

**Issue by obsy** [ericpaulbishop/gargoyle#504]
"Brak definicji "wifi_channel_width" (in polish) - english: lack of definition "wifi_channel_width".

line 2910, and only for IE10/IE11 (not edge) in Windows10. basic tab, default config. 

Line 2910: 	setChannelWidth(Object(wifi_channel_width),"G"); ...

**Comment by lantis1008** [comment]
This is only an issue in IE. I looked into fixing it but I went around in circles. 

The code as written in this repository works. 
IE stuffs it around and changes it to object code, and then it breaks. 

It's another case of Internet explorer monkeys being monkeys.

**Comment by lantis1008** [comment]
This is a relevant article on the issue. 
http://www.impressivewebs.com/avoiding-problems-with-javascript-getelementbyid-method-in-internet-explorer-7/

**Comment by nworbnhoj** [comment]
IMHO the only worthwhile effort relating to IE is to (help people) change to another browser (Firefox)

**Comment by lantis1008** [comment]
```
<script type="text/javascript">
if (/msie/i.test (navigator.userAgent)) //only override IE
{
	document.nativeGetElementById = document.getElementById;
	document.getElementById = function(id)
	{
		var elem = document.nativeGetElementById(id);
		if(elem)
		{
			//make sure that it is a valid match on id
			if(elem.attributes['id'].value == id)
			{
				return elem;
			}
			else
			{
				//otherwise find the correct element
				for(var i=1;i<document.all[id].length;i++)
				{
					if(document.all[id][i].attributes['id'].value == id)
					{
						return document.all[id][i];
					}
				}
			}
		}
		return null;
	};
}
</script>
```

This is the suggested workaround. I can't see the code offending any other criterion listed in that article?

**Comment by ericpaulbishop** [comment]
If this is happening in the latest version of IE, that's a pretty big issue. If it's happening in an ancient version of IE, not so much.  I'm curious whether the above code snippet actually fixes the issue, and just how widespread the problem is. If it's just a matter of including that code, it's easy to add in common.js. On the other hand, if this is really rare, it might not be worth it.

**Comment by lantis1008** [comment]
It definitely occurs in IE11. 
I'll test that code snippet. I had the same thought that it should go in common.js

**Comment by lantis1008** [comment]
I don't have the means to test in any IE version other than 11.
Here are my findings:
1. IE11 is no longer easily identifiable by useragent probing due to them dropping the tag "MSIE". It is the only browser I can identify that doesn't make itself known easily. Brilliant. :-1: 
2. This doesn't work on IE11 because this isn't the actual problem. IE is interpreting the javascript, and converting the line ```setChannelWidth(document.getElementById("wifi_channel_width"),"G")``` to ```setChannelWidth(Object(wifi_channel_width),"G")``` which errors out as "wifi_channel_width undefined". 
3. This line and the one below it are the ONLY ones that internet explorer has decided to convert to another format.

**Comment by lantis1008** [comment]
I'm a fool. 
Will patch as a part of #501

**Comment by lantis1008** [comment]
Resolved by 1f4726222fb9a0c1f2b8a6227ae8cedf73ca0d98

**Comment by obsy** [comment]
Yep, looks fine.
